hw/ip/otbn/doc/_index.md

```diff
@@ -565,6 +564,12 @@ The error that caused the alert can be determined by reading the {{< regref "FAT
 If OTBN was running, this value will also be reflected in the {{< regref "ERR_BITS" >}} register.
 A fatal alert can only be cleared by resetting OTBN through the `rst_ni` line.
 
+### Reaction to Life Cycle Escalation Requests
+
+OTBN receives and reacts to escalation signals from the [life cycle controller]({{< relref "/hw/ip/lc_ctrl/doc#security-escalation" >}}).
+
+An escalation request signaled through the `lc_escalate_en_i` signal results in the same action as a [fatal error](#design-details-fatal-errors) without raising a fatal alert.
```

Comment:
Maybe "... but does not raise a fatal alert"? I think it's a bit clearer that way?